pyspectre/core.py

```diff
@@ -75,33 +78,34 @@ def simulate( netlist_path: str, includes: Iterable[str] = None
 
     #ret = os.system(cmd)
     ret = run( cmd
-             , check          = False
-             , stdin          = DEVNULL
-             , stdout         = DEVNULL
-             , stderr         = DEVNULL
-             , capture_output = False
+             , check         =True
+             , stdin         =DEVNULL
+             , stdout        =True
+             , stderr        =True
+             , capture_output=False
              , ).returncode
 
     if ret != 0:
-        if log_path:
-            with open(log_path, 'r', encoding = 'utf-8') as log_handle:
+        if log_path is not None:
```

Comment:
A string is a truthy value, no? Has this ever failed `if log_path` ever produced wrong results?

Reply:
I wasn't sure maybe while debugging I should have reverted back.